pycep/transformer.py

```diff
@@ -898,46 +898,36 @@ def extension_resource_id(self, args: list[Token]) -> pycep_typing.ExtensionReso
             },
         }
 
-    def list_keys(
+    def list_func(
         self,
         args: tuple[
             pycep_typing.PossibleValue,
             pycep_typing.PossibleValue,
-            pycep_typing.PossibleNoneValue,
-            pycep_typing.PossibleNoneValue,
+            pycep_typing.PossibleValue,
         ],
-    ) -> pycep_typing.ListKeys:
-        resource_identifier, api_version, property_name_1, property_name_2 = args
+    ) -> pycep_typing.ListFunc:
+        func_name, resource_identifier, api_version = args
 
-        result: pycep_typing.ListKeys = {
-            "type": "list_keys",
+        result: pycep_typing.ListFunc = {
+            "type": str(func_name),
             "parameters": {
                 "resource_identifier": resource_identifier,
                 "api_version": api_version,
             },
```

Comment:
```suggestion
            "type": "list_func"
            "parameters": {
                "func_name": str(func_name),
                "resource_identifier": resource_identifier,
                "api_version": api_version,
            },
```
this would fit the `TypedDict` you defined for `ListFunc